docs/src/main/sphinx/object-storage/metastores.md

```diff
@@ -577,7 +578,6 @@ The Nessie catalog does not support [view management](sql-view-management) or
 [materialized view management](sql-materialized-view-management).
 
 (iceberg-snowflake-catalog)=
-
```

Comment:
revert?

Reply:
no ... these empty lines are redundant and we want to remove them